components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -1275,18 +1275,23 @@ private static void buildOpenIDConnectAuthenticationConfiguration(IdentityProvid
         property.setValue(paramMap.get("tokenUrl"));
         properties[2] = property;
 
+        property = new Property();
+        property.setName(IdentityApplicationConstants.Authenticator.OIDC.OIDC_LOGOUT_URL);
+        property.setValue(paramMap.get("logoutUrlOidc"));
```

Comment:
Check whether "logoutUrlOidc" value is correct as with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/53